### ISSUE TITLE
Latest Totals | Dialog | Close button needs aria-label

### ISSUE THREAD

**Issue by LaurieSReynolds** [COVID19Tracking/website#1694]
The close button currently announces "x" button - it should have an aria-label="close"

Resize the browser so the avg map is a panel of state squares instead of hexagons. 
Selecting a state brings up the latest totals panel

![image](https://user-images.githubusercontent.com/18370991/102686427-8f4fb000-419c-11eb-8ca0-46a68cbced67.png)

![image](https://user-images.githubusercontent.com/18370991/102686454-b6a67d00-419c-11eb-8f3c-faec0d187e8c.png)
